python/ray/services.py

```diff
@@ -505,8 +500,10 @@ def start_redis(node_ip_address,
             addresses for the remaining shards, and the processes that were
             started.
     """
-    redis_stdout_file, redis_stderr_file = new_redis_log_file(redirect_output)
 
+    assert len(redirect_files) == 1 + num_redis_shards, (
```

Comment:
This should raise a `ValueError` instead of an `assert` (see other comment with the rationale). Could you change this in a subsequent PR?